### PR TITLE
added action tag

### PR DESCRIPTION
**Pull request by mynameiswhm**

### CODE REVIEW

File: aggregator/publisher.go [hunk 1]
```diff
@@ -69,10 +69,10 @@ func (w *Writer) Start() {
 						continue // no server tag :(
 					}
 
-					tags := m.Tags.Filter(&[]string{"status", "user", "type", "region"})
+					tags := m.Tags.Filter(&[]string{"status", "user", "action", "type", "region"})
```
Comment (olegfedoseev) [inline aggregator/publisher.go]: Maybe "category" or "controller" is more suitable? After all we usually have "action" in "script".

Comment (mynameiswhm) [inline aggregator/publisher.go]: the idea was "use script if that's enough for you, or put class name into script, and action into, well, action".

so there's two options:
- script=Some_Class_Name, action=list
- category=Some_Class_Name, script=Some_Class_Name::list

Comment (olegfedoseev) [inline aggregator/publisher.go]: Second option allows more general approach, and that's the way i'd like it to be. In category you can set more generic part of application. 
script=Some_Class::_action category=Search
script=Some_Other_Class::_otherAction category=Item

Script and action are more limiting. And with first option we may lose backward compatibility.

**Comment by mynameiswhm** [comment]
changed action to category

**Pull request merged**